kairos-cli/bin/crypto/error.rs

```diff
@@ -0,0 +1,14 @@
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum CryptoError {
+    /// Unable to load a file from the given path.
+    #[error("failed to load key from file")]
+    KeyLoad,
+    /// Failed to parse a public key from a raw data.
+    #[error("failed to parse private key")]
+    FailedToParseKey,
```

Comment:
I don't love `casper_types::crypto::error::ErrorExt`, but we should use it anyway.

Reply:
Do you mean it's missing here?

Reply:
Yeah, it should be wrapped in a variant here.